services/.eslintrc

```diff
@@ -0,0 +1,5 @@
+---
+rules:
+  import/no-extraneous-dependencies: off
```

Comment:
In fact, I'd keep `import/no-extraneous-dependencies` on in handlers. The reason is that only `aws-sdk` is provided by the AWS environment, the other dependencies have to be packaged and sent to the lambda function. It is a good reminder sometimes

Reply:
That's why the ignore rules are added only for for files in the `/services` folder. The custom code in `/lib` will not be ignored for example.

Reply:
https://github.com/ec-europa/eubfr-data-lake/pull/9/files#diff-5882116b481bdd4f41d98c1e32b13749L3 is not in `/services` :)